tikv-client/src/main/java/com/pingcap/tikv/exception/TableNotExistException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.pingcap.tikv.exception;
+
+public class TableNotExistException extends RuntimeException {
+  private String databaseName;
+  private String tableName;
+
+  public TableNotExistException(String databaseName, String tableName) {
```

Comment:
What is the message looks like? Should we format a message here?

Reply:
fix